packages/api-clients/open-api/delegationApi.yml

```diff
@@ -160,6 +160,52 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /delegations/{delegationId}/contracts/{contractId}:
+    get:
+      description: Retrieves a delegation contract
+      summary: Retrieves a delegation contract
+      tags:
+        - delegation
+      operationId: getDelegationContract
+      parameters:
+        - $ref: "#/components/parameters/CorrelationIdHeader"
+        - name: delegationId
+          in: path
+          description: The delegation id
+          required: true
+          schema:
+            type: string
+        - name: contractId
+          in: path
+          description: The delegation contract id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Delegation contract retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DelegationContractDocument"
+        "401":
```

Comment:
This error is not raised in the code nor mapped in the errorMapper, we can delete it from here

Reply:
I think it should stay, 401 is raised when the jwt is expired/not valid